jobs/e2e_node/containerd/ubuntu-init-cpu-manager.yaml

```diff
@@ -0,0 +1,13 @@
+#cloud-config
```

Comment:
IIUC this is derived from `ubuntu-init-hugepages-1G-allocation.yaml` which in turn seems a fixed and a stripped down version of `init.yaml`. Is that the case?
Two followup questions: do we need cni-plugins 0.7.5 or can we use 1.0.1? if we can use 1.0.1, can we just use `init.yaml`?

The reason I'm asking is because unless we have clearly different needs (like hugepages has!) I think is best to reuse existing common config and avoid proliferation of lane-specific config if we can help it, to keep the complexity at bay.

Reply:
Done.